Packs/DomainTools_Iris/Integrations/DomainTools_Iris/DomainTools_Iris.yml

```diff
@@ -3,6 +3,13 @@ commonfields:
   id: DomainTools Iris
   version: -1
 configuration:
+- display: DomainTools API URL
+  name: base_url
+  required: true
```

Comment:
i think setting it as mandatory will raise error on content installation for configured instances

Reply:
It's ok as there is a default value